packages/react/src/Bento/Bento.module.css

```diff
@@ -0,0 +1,1793 @@
+.Bento {
+  display: grid;
+  width: 100%;
+  grid-template-columns: repeat(12, minmax(0, 1fr));
+  grid-auto-rows: var(--base-size-80);
+  grid-gap: var(--brand-Bento-gap-small);
+  position: relative;
+}
+
+.Bento__Item {
+  background-color: var(--brand-Bento-bgColor);
+  border-radius: var(--brand-Bento-Item-borderRadius-small);
+  display: grid;
+  grid-template-rows: auto 1fr;
+  grid-template-columns: 1fr;
+  overflow: hidden;
+  border: 1px solid var(--brand-color-border-default);
+}
+
+@media screen and (min-width: 80rem) {
+  .Bento {
+    gap: var(--brand-Bento-gap-large);
+  }
+  .Bento__Item {
+    border-radius: var(--brand-Bento-Item-borderRadius-large);
+  }
+}
+
+@media screen and (min-width: 48rem) {
+  .Bento {
+    gap: var(--brand-Bento-gap-medium);
+  }
+  .Bento__Item {
+    border-radius: var(--brand-Bento-Item-borderRadius-medium);
+  }
+}
+
+/* Bento Content */
+.Bento__Item--visual-as-background .Bento__Content {
+  order: 1;
+  grid-row: 1 / span 2;
+  grid-column: 1 / span 2;
+}
+
+.Bento__Content {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  height: 100%;
+  color: var(--brand-color-text-default);
+}
+
+/* Column Span */
+.Bento__Item--column-span-1 {
```

Comment:
This is quite a lot of duplicate css from Grid we're shipping for column spanning. Curious if we explored using the grid internally for this component? [Here's the same styles from that component](https://github.com/primer/brand/blob/04790926c578c535cd39b410441ca8252180b2d8/packages/react/src/Grid/Grid.module.css#L37).

Reply:
I think we need to revisit this soon and find a way to consolidate some of the duplicate styles...

Having a quick look at the css bundle size increase:

Current: 411kb 
In this PR: 467kb 

Result: 13.6% increase being introduced in this PR.

We can internally refactor this after shipping it, but if there's a way to reuse the grid styles and bring this down that would be great 🙏